src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,50 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exeption.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMAL_LENGTH = 6;
```

Comment:
The constant name 'MINIMAL_LENGTH' could be more descriptive. Consider renaming it to 'MIN_PASSWORD_LENGTH' or 'MIN_LOGIN_LENGTH' to clarify what it represents.